scripts/docs/stringOfLength.js

```diff
@@ -0,0 +1,7 @@
+export default function stringOfLength(string, length) {
+  let newString = "";
+  for (let i = 0; i < length; i += 1) {
+    newString += string;
+  }
```

Comment:
could maybe use `for of` here too?

Reply:
same as https://github.com/penx/govuk-react/pull/79#discussion_r170896895

Reply:
ok